src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,59 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+type Props = {
+  handleStatusChange: (status: string) => void,
+  handleQueryChange: (query: string) => void,
+  resetQuery: () => void,
+  query: string,
+};
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+export const TodoFilter: React.FC<Props> = ({
+  handleStatusChange,
+  handleQueryChange,
+  resetQuery,
+  query,
+}) => {
+  return (
+    <form className="field has-addons">
+      <p className="control">
+        <span className="select">
+          <select
+            data-cy="statusSelect"
+            onChange={event => {
+              handleStatusChange(event.target.value);
```

Comment:
You could use this logic right in the handle
Here we could just put `onChange={handleStatusChange}`
And in the handler
`  const handleStatusChange = (event) => {
    setStatus(event.target.value);
  };`

Reply:
Good point